### DISCUSSION TITLE
Update Backend Form after adding relation

### DISCUSSION THREAD

**Discussion by thomas4Bitcraft** [wintercms/winter#215] [Q&A] [answered]
Is there a way to update the parent form when adding a relation with the relation manager? I have this form and want to update the chart if a new vote is created:

![](https://cdn.discordapp.com/attachments/821159568763125800/857898532428972032/unknown.png)

**Comment by mjauvin** [comment, marked as answer]
did you try to use fields dependencies (dependsOn) ?

ref. https://wintercms.com/docs/backend/forms#field-dependencies

**Reply by thomas4Bitcraft** [reply]
Not until now -> works - thank you!
